src/features/reservations.feature

```diff
@@ -0,0 +1,75 @@
+Feature: Reservation Management
+
+Background:
+    Given User navigates to the application
+    And User logs in
+
+# Scenario:  Editing the "No-Show" Settings
+#     Given that the user is on the reservation page
+#     When the user clicks on the "Automatically Mark No-Show" checkbox
+#     And the user adjusts the time duration in the checkbox
+#     And the user clicks on 'Save' button
+#     Then the modified data should be saved 
+
+
+Scenario Outline: Verify that the admin user can add, edit, and delete a Schedule
+        # Add Schedule
+    Given that the user is on the reservation page
+    Then the user clicks on the "+" button within the "Schedule" section
+    When the user fills in the required “Name” input
+    And the user selects the calendar Start Date
+    And the user selects the calendar End Date
+    And the user clicks on the "Start when course opens" checkbox
+    And the user clicks on the "End when course closes" checkbox
+    And the user selects the "Start Time"
+    And the user selects the "End Time"
+    And the user and selects an option from the repeat dropdown
```

Comment:
If user can select just one option from dropdown,  no sense to use datatable

Reply:
done. I got rid of the table.